go.mod

```diff
@@ -26,18 +23,14 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.0.0
 	github.com/wadey/gocovmerge v0.0.0-20160331181800-b5bfa59ec0ad // indirect
-	golang.org/x/time v0.0.0-20190921001708-c4c64cad1fd0 // indirect
-	google.golang.org/appengine v1.6.5 // indirect
-	google.golang.org/genproto v0.0.0-20190916214212-f660b8655731 // indirect
-	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
-	istio.io/api v0.0.0-20200724154434-34e474846e0d
-	istio.io/client-go v0.0.0-20200708142230-d7730fd90478
-	k8s.io/api v0.18.1
-	k8s.io/apimachinery v0.18.1
-	k8s.io/client-go v0.18.1
+	istio.io/api v0.0.0-20201112235759-fa4ee46c5dc2
+	istio.io/client-go v0.0.0-20200908160912-f99162621a1a
+	k8s.io/api v0.19.3
+	k8s.io/apimachinery v0.19.3
+	k8s.io/client-go v0.19.3
 )
 
 replace (
-	github.com/golang/protobuf => github.com/golang/protobuf v1.3.2
-	k8s.io/client-go => k8s.io/client-go v0.18.0
+	github.com/golang/protobuf => github.com/golang/protobuf v1.4.2
+	k8s.io/client-go => k8s.io/client-go v0.19.3
```

Comment:
if the versions stay after `make` is run, these can be removed

Reply:
that didn't work. 
../../../../pkg/mod/k8s.io/client-go@v0.18.0/kubernetes/scheme/register.go:27:2: module k8s.io/api@latest found (v0.20.1), but does not contain package k8s.io/api/auditregistration/v1alpha1

Reply:
sounds like a protobuf issue. what happens when you only remove line 35 with the k8s api?